mvc/controllers.js

```diff
@@ -37,10 +38,29 @@ exports.getArticleByIdController = (req, res, next) => {
 
 exports.getCommentsByArticleIdController = (req, res, next) => {
     const { article_id } = req.params
-    const promises = [getCommentsByArticleIdModel(article_id), checkCategoryExists(article_id)]
+    const promises = [
+        getCommentsByArticleIdModel(article_id),
+        checkCategoryExists(article_id),
+    ]
     Promise.all(promises)
         .then(([comments]) => {
             res.status(200).send({ comments })
         })
         .catch(next)
 }
+
+exports.postCommentController = (req, res, next) => {
+    const {
+        body,
+        params: { article_id },
+    } = req
+    const promises = [
+        checkCategoryExists(article_id),
+        postCommentModel(body, article_id),
+    ]
+    Promise.all(promises)
+        .then(([firstPromise, comment]) => {
```

Comment:
As you're ignoring the first variable in the resolved array you could use `[_, comment]` to denote that the first variable won't be used.